docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3298,6 +3305,68 @@ entries. When a priority value is required (&eg; for tables including `RANGE`,
 `TERNARY` or `OPTIONAL` matches), it is inferred based on the order in which
 entries appear in the table declaration.
 
+TODO: With the recent change to the P4~16~ specification for initial
```

Comment:
Any further insights into this concern?

Reply:
Here is the proposed PR for the language spec that would make it explicit that it is a compile-time error to try to specify priorities in the source code within `const entries`, matching the current open source p4c behavior: https://github.com/p4lang/p4-spec/pull/1259/files

Reply:
With commit 11 of this PR, I have removed the paragraph marked TODO, and replaced it with a footnote linking to this PR on the language spec, which is not accepted and merged in yet, but I'm being optimistic about that: https://github.com/p4lang/p4-spec/pull/1259